swap_meet/electronics.py

```diff
@@ -1,2 +1,9 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+    
+    def __init__(self, condition = 0.0):
+        super().__init__(category="Electronics", condition = condition)
+
+    def __str__(self):
+        return "A gadget full of buttons and secrets."
```

Comment:
Great work in setting up all the child classes (electronics, clothing, and decor). Especially having category set as a default parameter in the parent constructor. This prevents users from changing the category value for Electronic instances by accident. (The same goes for decor and clothing instances as well).